test/CheckoutSdkTest/Risk/RiskIntegrationTest.cs

```diff
@@ -117,9 +117,9 @@ private async Task TestRequestPreAuthenticationRiskScan(RiskPaymentRequestSource
                 .RequestPreAuthenticationRiskScan(preAuthenticationAssessmentRequest);
 
             response.ShouldNotBeNull();
-            response.AssessmentId.ShouldNotBeNull();
-            response.Result.ShouldNotBeNull();
-            response.Result.Decision.ShouldNotBeNull();
+            // response.AssessmentId.ShouldNotBeNull();
```

Comment:
this could be deleted instead of commented, you intend to use it in the future ?

Reply:
Fix, thank you